README.md

```diff
@@ -115,10 +115,14 @@ sudo npm -g install grunt-cli
 Ubuntu/Mac:
 ```
 sudo npm -g install jspm
-jspm endpoint config github
+jspm registry config github
```

Comment:
OK, might be worth sending out an email when this goes out to let devs know they will need to do the step below.